applications/chats/urls.py

```diff
@@ -1,10 +1,11 @@
 from django.urls import include, path
 from rest_framework.routers import DefaultRouter
 
-from .views import RoomViewSet
+from .views import MessageViewSet, RoomViewSet
 
 
 router = DefaultRouter()
+router.register(r'(?P<room>[0-9]+)/messages', MessageViewSet, basename='message')
```

Comment:
why are the regexes needed?

Reply:
I want the url match with the format `/api/v1/chats/:room/messages/`. The regex ensures that `room` value is a possible number id.